recipe/meta.yaml

```diff
@@ -84,25 +84,34 @@ outputs:
       binary_relocation: False
       detect_binary_files_with_prefix: False
     requirements:
-{% if cross_target_platform == "linux-64" %}
       run:
+{% if cross_target_platform == "linux-64" %}
+        # on linux-64 2.12 has the highest version and 2.17/2.28 are lower
         - _sysroot_{{ cross_target_platform }}_curr_repodata_hack 3.*
+{% else %}
+        # on non-x86_64, 2.17 has the highest version
+        - _sysroot_{{ cross_target_platform }}_curr_repodata_hack 4.*
 {% endif %}
       run_constrained:
         - sysroot_{{ cross_target_platform }} =={{ version }}
     test:
       commands:
         test -f $PREFIX/{{ target_machine }}-{{ ctng_vendor }}-linux-gnu/sysroot/usr/include/linux/version.h
 
-{% if cross_target_platform == "linux-64" %}
   - name: _sysroot_{{ cross_target_platform }}_curr_repodata_hack
+{% if cross_target_platform == "linux-64" %}
+    # on linux-64 2.12 has the highest version and 2.17/2.28 are lower
```

Comment:
This would be
- 4 -> 2.12
- 3 -> 2.17
- 2 -> 2.28
?

Reply:
We only need the one we want to be the default to be on v4 and the other ones, older or newer, on v3. So for 2.28 we will have v3 as well.